packages/flutter/lib/src/widgets/form.dart

```diff
@@ -291,18 +291,44 @@ class FormState extends State<Form> {
   /// returns true if there are no errors.
   ///
   /// The form will rebuild to report the results.
+  ///
+  /// See also:
+  ///  * [validateGranularly], which returns a [Map] describing the validation
```

Comment:
Looks like it's no longer a `Map`?

Reply:
"which also validates descendant FormFields, but instead returns fields with errors" or something similar?